.github/workflows/release.yml

```diff
@@ -22,6 +18,15 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20
+      - name: Update npm version
```

Comment:
If we already had `pre-release.yml` and write back update npm version to the repo, I think we could get rid of the process here?

Reply:
This one seems to be still here?

Reply:
Huh, swore I removed that. Fixed.